packages/components/src/Banner/Banner.stories.tsx

```diff
@@ -15,186 +15,203 @@ export default {
       },
     },
   },
+  args: {
+    content:
+      "Summit Learning has a full-time team dedicated to constantly improving our curriculum. To see the updates, click into the course.",
+    heading:
+      "New curriculum updates are available for one or more of your courses.",
+  },
```

Comment:
Top level args are inherited by each story.

We could always do this, but we used to lint against it due to limitations of `prepareStory`.